### PR TITLE
fix broken links in IA Spec

### LINKED ISSUE #249: Links broken in IA Spec
On this page: [Who has to use the Canada.ca design sytem - Canada.ca]
(https://design.canada.ca/architecture/usage-canadaca-design.html)

This link gives a 404: 

**These organizations**
These organizations must apply Canada.ca design requirements for all their public-facing websites or digital services, regardless of the technology, domain name (both canada.ca and gc.ca) or publishing platform through which these are delivered.

The ‘Previous’ and ‘Next’ buttons are also missing from the bottom of the page. 

On this page: [Mandatory elements of the Canada.ca design system - Canada.ca](https://design.canada.ca/architecture/mandatory-elements.html)

The ‘Previous’ and ‘Next buttons at the bottom of the page are giving 404s

On this page: [Organizing content on Canada.ca - Canada.ca](https://design.canada.ca/architecture/organizing-content.html)

This link gives a 404: 

**Canada.ca topic tree** 
See where your content fits under themes and their corresponding topics in the Canada.ca Topic tree.

### PR DESCRIPTION
**Pull request by jmealing**
Fixing: https://github.com/canada-ca/design-system/issues/249

FR: https://github.com/canada-ca/systeme-conception/pull/147

Preview: https://deploy-preview-250--design-system-canada-ca.netlify.app/architecture/canada-content-information-architecture-specification.html

### CODE REVIEW

File: architecture/mandatory-elements.md
Comment (delisma) [inline architecture/mandatory-elements.md:1]: Should we apply this patch to the French equivalent as well?

**Review by delisma — CHANGES_REQUESTED** [review]

File: architecture/mandatory-elements.md
Comment (jmealing) [reply architecture/mandatory-elements.md:1]: I see the page navigation on the bottom is here already. Only the FR was missing it. The EN just needed links updated.

**Review by delisma — APPROVED** [review]
Pre-approved. Waiting for the alternate language PR to be approved too.

**Comment by prycrane** [comment]
I fixed the alignment on canada-content-information-architecture-specification.html

**Pull request merged**